x-pack/legacy/plugins/siem/index.ts

```diff
@@ -135,6 +135,17 @@ export const siem = (kibana: any) => {
         },
       },
       mappings: savedObjectMappings,
+      // TODO: Remove once while Saved Object Mappings API is programmed for the NP See: https://github.com/elastic/kibana/issues/50309
+      savedObjectSchemas: {
+        'case-workflow': {
+          indexPattern: '.case-testing-ground', // TODO: Change this name and use kibana.yml settings to override it.
```

Comment:
What's the reason for having this stored in a separate index? If we can avoid it, we should as a failed migration requires each of these indices to to removed currently and won't be resolved until 8.0.

Reply:
i will need it in a separate index, but apparently there is a way to get access to the `kibana.yml` from this part of the code which I had not believed to be possible. I need to track down an example, brb!

Reply:
I removed these lines for now since they will need to be done from the NP side ultimately. could have to do with these failures, we'll see